sample/src/androidTest/kotlin/com/agoda/sample/TestActivityTest.kt

```diff
@@ -67,7 +67,7 @@ class TestActivityTest {
                 action {
                     hasText("DISMISS")
                     click()
-                    idle(500)
+                    this@screen.idle(500)
```

Comment:
why this change?

Reply:
After correct implementation of `DslMarker`, `idle` function is outside of the scope so you need to explicitly access screen to idle. Maybe we can think of something to make this function publicly accessible across all dsl? Maybe put it to `Screen`'s companion object?

Reply:
Moved `idle` to `Screen`'s companion object.